common/lib/dependabot/errors.rb

```diff
@@ -76,6 +76,20 @@ def initialize(source, msg = nil)
   # File level errors #
   #####################
 
+  class ToolVersionNotSupported < DependabotError
+    attr_reader :tool_name, :detected_version, :supported_versions
+
+    def initialize(tool_name, detected_version, supported_versions)
+      @tool_name = tool_name
+      @detected_version = detected_version
+      @supported_versions = supported_versions
+
+      msg = "Dependabot detected the following #{tool_name} requirement for your project: '#{detected_version}'." \
+            "\n\nCurrently, the following Python versions are supported in Dependabot: #{supported_versions}."
```

Comment:
should this be `tool` instead of the hardcoded Python?

Reply:
Oh wow, nice catch! Let me update it!